lib/rules/block-opening-brace-space-after/index.js

```diff
@@ -38,13 +39,22 @@ function rule(expectation, options, context) {
 			],
 		});
 
-		if (!validOptions) {
+		if (
+			!validOptions ||
+			optionsMatches(options, 'ignore', 'rules') ||
+			optionsMatches(options, 'ignore', 'at-rules')
+		) {
 			return;
 		}
 
-		// Check both kinds of statements: rules and at-rules
-		root.walkRules(check);
-		root.walkAtRules(check);
+		// Check both kinds of statement: rules and at-rules
+		if (!optionsMatches(options, 'ignore', 'rules')) {
```

Comment:
The original issue was to only add ignore: ["at-rules"] to this rule. We should revert this change.